packages/apps/foundationdb/Chart.yaml

```
@@ -0,0 +1,25 @@
+type: application
+
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+# Versions are expected to follow Semantic Versioning (https://semver.org/)
+version: 0.1.0
+
+# This is the version number of the application being deployed. This version number should be
+# incremented each time you make changes to the application. Versions are not expected to
+# follow Semantic Versioning. They should reflect the version the application is using.
+# It is recommended to use it with quotes.
+appVersion: "7.4.1"
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)